css/style.css

```diff
@@ -664,6 +1572,67 @@
     flex-direction: column;
     padding: 11px 0 55px 45px;
   }
+
+  .wrap-information, .wrap {
+    height: 873px;
+  }
+
+  .info__title {
+    font-size: 56px;
+    text-align: right;
+    padding: 0 45px 30px 0;
+    margin-top: 20px;
+  }
+
+  .red-first, .details {
+    height: 477px;
+  }
+
+  .red-first {
+    padding-left: 62px;
+  }
+
+  .description__title, 
+  .description__title-most {
+    font-size: 43px;
+    letter-spacing: 3px;
+  }
+
+  .drive, 
+  .model-s {
+    font-size: 21px;
+    padding: 25px 25px 64px 0;
+  }
+
+  .icons, .info-order, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)